<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ffmpegExport</class>
 <widget class="QWidget" name="ffmpegExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="mp4RadioBtn">
           <property name="text">
            <string>MP4 H264</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="aviRadioButton">
           <property name="text">
            <string>AVI MPEG-4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="vmwRadioBtn">
           <property name="text">
            <string>WMV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="gifRadioBtn">
           <property name="text">
            <string>Gif</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Frames per second</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fpsSpinbox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="selectFolderBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Select Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="folderLabel">
        <property name="text">
         <string>C:/</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportVideoBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
